### DISCUSSION TITLE
Is getShopByOwnerUserId working as intended?

### DISCUSSION THREAD

**Discussion by xaviervc** [etsy/open-api#1249] [Q&A] [answered]
If the user id path parameter mismatches with the Authorization Bearer token (which contains a user id in it), it will not return the right store but return what the Authorization Bearer token contains.

To test it out, just put a random int in the user_id path parameter and you'll get whatever store is attached to the bearer token instead of what the path parameter is.

**Comment by Gareth064** [comment, marked as answer]
This a badly documented and deceiving endpoint.

It will only return the shop of the userId which is in the Auth Token.

There is no way to get the Shop owned by a Seller who isn't using your App.

**Reply by xaviervc** [reply]
Ok that's fine, I can work with the findShops endpoint, thanks for the reply.
